media/home/js/main.js

```diff
@@ -60,4 +60,63 @@ document.addEventListener("DOMContentLoaded", () => {
     xhttp.send(params);
   };
 
+  const name = document.getElementsByName('name');
+  const email = document.getElementsByName('email');
+  const message = document.getElementsByName('message');
+  const send = document.getElementsByName('send');
+
+  if (document.getElementsByName('send')) {
+    send.addEventListener("click", () => {
+      if(name.value == "" || email.value == "") {
+        // window.location.href = baseUrl + 'login';
+        console.log('Required fields.');
+        console.log('Login Aborting...');
+        iitpConnect.renderMessage('Required fields', 'error',5000 );
+      } else {
+        setApp();
+      }
+
+    });
+  }
+
+  const setApp = () => {
+    iitpConnect.startLoader();
+
+    const xhttp = new XMLHttpRequest();
+    const url = baseUrl + '/index.php';
+    const params = 'submit=' + '&token=' + tok.value + '&username=' + name.value + '&userpassword=' + email.value +  '&message=' + message.value + '&task=User.sendFeedback';
```

Comment:
The task should be controller class name like home controller `HomeController.sendFeedback`. Do not try to call libraries to function directly. Call first in controller class of view.